### DISCUSSION TITLE
[Question] How to Use Unique Cookie Names in Development

### DISCUSSION THREAD

**Discussion by jneander** [nextauthjs/next-auth#2311] [Help] [answered]
I have two next apps running locally, mostly unrelated to each other. They both use next-auth. However, next-auth is confusing the cookies between the apps due to them sharing the same names on the same domain (localhost). When one server tries to read the cookie and fails, it unsets the cookie and causes both apps to be signed out.

How can two or more next apps using next-auth run locally without conflicting with each other?

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/configuration/options#cookies

See the defaults here: https://github.com/nextauthjs/next-auth/blob/8bbffdd08c6e9e125fca2e5aaa29b77d98cf4ef0/src/server/lib/cookie.js#L137

**Reply by jneander** [reply]
Excellent! Works perfectly. Thank you. Note: I am heeding the guidance and using this only in development.

Question: Has this come up before? I looked for discussions before creating this one, but turned up nothing.
